src/app/user/components/create-profile/create-profile.component.ts

```diff
@@ -0,0 +1,122 @@
+import { Component, OnInit } from '@angular/core';
+import { FormGroup, FormBuilder, Validators } from '@angular/forms';
+import { Router, ActivatedRoute } from '@angular/router';
+import { UserType } from 'src/app/shared/models/userType';
+import { AngularFirestore, AngularFirestoreCollection, AngularFirestoreDocument } from '@angular/fire/firestore';
+import { User } from 'firebase';
+import { UserProfile } from 'src/app/shared/models/userProfile';
+
+@Component({
+  selector: 'app-create-profile',
+  templateUrl: './create-profile.component.html',
+  styleUrls: ['./create-profile.component.less']
+})
+export class CreateProfileComponent implements OnInit {
+  companyUserType: UserType;
+  normalUserType: UserType;
+  userType?: UserType;
+  firstName: string;
+  lastName: string;
+  companyName?: string;
+  birthDate: Date;
+  interests?: string;
+  workExperience?: string;
+  educationalExperience?: string;
```

Comment:
what are those above variables used for? (from firstName to educationalExperience)